miqa/core/conversion/import_export_csvs.py

```diff
@@ -33,7 +38,7 @@ def validate_file_locations(input_dict, project):
     return input_dict
 
 
-def validate_import_dict(import_dict, project: Project):
+def validate_import_dict(import_dict, project):
```

Comment:
I'd suggest we change this to 
```suggestion
def validate_import_dict(import_dict, project: Optional[Project]):
```

And have global status indicated by `project is None`.

Reply:
changed in f932555